host/xtest/ffa_spmc_1000.c

```diff
@@ -137,10 +121,12 @@ static void xtest_tee_test_9001(ADBG_Case_t *c)
 	}
 
 	memset(&args, 0, sizeof(args));
-	ADBG_EXPECT_TRUE(c, start_sp_test(endpoint1_id, EP_TEST_SP, &args));
-	ADBG_EXPECT_TRUE(c, check_ok(&args));
+	ADBG_REQUIRE_TRUE(c, exit_test, start_sp_test(endpoint1_id, EP_TEST_SP,
+			  &args))
+	ADBG_REQUIRE_TRUE(c, exit_test, check_ok(&args))
+	Do_ADBG_EndSubCase(c, NULL);
```

Comment:
While a subcase can be ended with a NULL pointer, it's preferable to supply the same name that was used when the subcase was opened. It's easier to see which subcase you're ending then.

Reply:
Ack